packages/jellyfish-transaction-builder/src/txn/txn_builder_loans.ts

```diff
@@ -105,123 +105,127 @@ export class TxnBuilderLoans extends P2WPKHTxnBuilder {
   }
```

Comment:
Looks like the only change here supposed to be all the @Deprecated in comment, all the changes are due to unnecessary order changes

Reply:
yup, the change is only the @deprecated and comment🥲

Reply:
I'll try to revert this